lib/readline.js

```diff
@@ -110,8 +110,13 @@ const {
 
 const { StringDecoder } = require('string_decoder');
 
-// Lazy load Readable for startup performance.
-let Readable;
+const { Readable } = require('stream');
```

Comment:
This undoes lazy loading for startup perf. That seems out of scope for a typings change that should be a comments-only change. Is there no way to add typings without altering the executing code?

Reply:
There is a way to import them without altering the code, I forgot to import `Readable` as a `typedef` just like `Writable` 😅